subsys/mgmt/hawkbit/hawkbit.c

```diff
@@ -1099,7 +1099,10 @@ static bool send_request(struct hawkbit_context *hb_context, enum hawkbit_http_r
 	static const char *const headers[] = {AUTH_HEADER_FULL, NULL};
 #endif /* CONFIG_HAWKBIT_SET_SETTINGS_RUNTIME */
 #endif /* CONFIG_HAWKBIT_DDI_NO_SECURITY */
-
+#ifdef CONFIG_HAWKBIT_SAVE_PROGRESS
+	char header_range[RANGE_HEADER_SIZE] = {0};
+	char const *headers_range[] = {header_range, NULL};
```

Comment:
had to put the const that way, checkpatch would let this fail otherwise
```
WARNING:STATIC_CONST_CHAR_ARRAY: char * array declaration might be better as static const
```
but that is meant for literal strings